projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/omsa-real-time-stock/omsa-real-time-stock-in-PDP.e2e-flaky.cy.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Lets remove omsa from file name as we have not used it anywhere else.

Reply:
Done